kong/plugins/prometheus/exporter.lua

```diff
@@ -39,21 +40,24 @@ local function init()
   prometheus = require("kong.plugins.prometheus.prometheus").init(shm, "kong_")
 
   -- global metrics
-  if kong_subsystem == "http" then
-    metrics.connections = prometheus:gauge("nginx_http_current_connections",
-      "Number of HTTP connections",
-      {"state"})
-  else
-    metrics.connections = prometheus:gauge("nginx_stream_current_connections",
-      "Number of Stream connections",
-      {"state"})
-  end
+  metrics.connections = prometheus:gauge("nginx_current_connections",
+    "Number of connections by subsystem",
+    {"node_id", "subsystem", "state"})
+  metrics.nginx_requests_total = prometheus:counter("nginx_requests_total",
+      "Number of requests total", {"node_id", "subsystem"})
+  metrics.nginx_requests_accepted = prometheus:counter("nginx_requests_accepted",
+      "Number of requests accepted", {"node_id", "subsystem"})
+  metrics.nginx_requests_handled = prometheus:counter("nginx_requests_handled",
+      "Number of requests handled", {"node_id", "subsystem"})
   metrics.timers = prometheus:gauge("nginx_timers",
                                     "Number of nginx timers",
                                     {"state"})
   metrics.db_reachable = prometheus:gauge("datastore_reachable",
                                           "Datastore reachable from Kong, " ..
                                           "0 is unreachable")
+  metrics.node_info = prometheus:gauge("node_info",
```

Comment:
For future use cases, should we actually add `ip` here at least as well? I don't think that it would introduce high-cardinality in this case since for a given node_id, the IP might always be the same correct?

Reply:
I think the thought here was to rely on some already default/automatically applied prometheus metrics/labels from the scrape to get that information